packages/react-accordion/src/components/Accordion/useAccordion.ts

```diff
@@ -1,51 +1,38 @@
 import * as React from 'react';
-import { makeMergeProps, resolveShorthandProps } from '@fluentui/react-utilities';
-import { AccordionProps, AccordionShorthandPropsCompat, AccordionState } from './Accordion.types';
+import { resolveShorthand } from '@fluentui/react-utilities';
+import { AccordionProps, AccordionSlots, AccordionState } from './Accordion.types';
 import { useCreateAccordionContextValue } from './useAccordionContext';
 
 /**
  * Const listing which props are shorthand props.
  */
-export const accordionShorthandPropsCompat: AccordionShorthandPropsCompat[] = ['expandIcon', 'button', 'icon'];
-
-const mergeProps = makeMergeProps<AccordionState>({ deepMerge: accordionShorthandPropsCompat });
+export const accordionShorthandProps: Array<keyof AccordionSlots> = [];
```

Comment:
is this supposed to be an empty array ?

Reply:
especially since you resolve the slots later in the hook

Reply:
Yes! the shorthands `Accordion` receives will not be used in the rendering of the `Accordion` itself, I use it only as mechanism to generate defaults that will be passed down the road trhough Context

Reply:
I'm not entirely of passing slots through contexts, but let's discuss that offline, it doesn't change much in this PR